spec/features/customers/orders/show_spec.rb

```diff
@@ -19,7 +19,7 @@
       end
 
       expect(page).to have_content "Status: #{order_3.status_name}"
-      expect(page).to have_content "$#{order_3.total_price_in_dollars}"
+      expect(page).to have_content order_3.total_price_in_dollars
       expect(page).to have_content "submitted at: #{order_3.display_submitted_at}"
```

Comment:
Also there was a strange issue with the order show_spec. It was passing sometimes and not passing other times. I suspect the issue is because when it was not passing, the total was always over $1,000, which makes me think the test was probably looking for "$1000" without the comma.

That is why I changed this line. It's not the best solution, so if someone has a better idea, please chime in!